src/layout/Stepper/Stepper.tsx

```diff
@@ -19,12 +24,25 @@ export interface StepperProps extends PropsWithChildren {
    * Style of the stepper. Default is dots but it can be numbered too.
    */
   variant?: 'default' | 'numbered';
+
+  /**
+   * Display link after last step
+   */
+  continuously?: boolean;
+
+  /**
+   * Animate items appearance
+   */
+  animated?: boolean;
```

Comment:
I've been thinking of allowing passing animated to a lot of our components and exposing the ability to pass the animation instructions. TBD tho. This is fine for now.